src/main/webapp/app/entities/feedback/feedback.model.ts

```diff
@@ -2,6 +2,14 @@ import { BaseEntity } from 'app/shared';
 import { Result } from '../result';
 import { ElementType } from '@ls1intum/apollon';
 
+export const enum FeedbackHighlightColor {
```

Comment:
One more thing:

I think it is never clearly defined, what kind of feedback is marked in what color.
It is only implicitly through code.

Maybe it would be better to structure the FeedbackHighlightColor enum differently:

`export enum FeedbackHighlightColor {
    CONFLICT = 'rgba(219, 53, 69,0.6)', // red
    ....
}`

Reply:
I thought about it and wanted to structure it like this at first. The reason I implemented it the way it is now is that we e.g. use the red color for multiple purposes (conflict and missing elements). So the name `CONFLICT` would not be 100% correct. If you think it would be better anyway, I can change it though.

Reply:
I understand, thanks for clarifying! Maybe we could have multiple enum keys with the same color?

Reply:
As discussed, we will keep it like this for now as I am not happy with different enum keys for the same color either.